RetailCoder.VBE/UI/RubberduckUI.sv.resx

```diff
@@ -1123,4 +1123,7 @@ Varning: Alla anpassade inställningar kommer att försvinna.    Din gamla fil k
   <data name="RenamePresenter_TargetIsInterfaceMemberImplementation">
     <value />
   </data>
-</root>
+  <data name="SourceControl_DeleteBranchButton" xml:space="preserve">
+    <value>Ta bort gren</value>
```

Comment:
@Zomis they're all Git commands under the hood. So, I suspect it does exactly what you think it does.